utils/check_repo.py

```diff
@@ -47,6 +47,7 @@
 # Being in this list is an exception and should **not** be the rule.
 IGNORE_NON_TESTED = PRIVATE_MODELS.copy() + [
     # models to ignore for not tested
+    "MaskFormerSwinBackbone",  # Backbones have their own tests.
     "ResNetBackbone",  # Backbones have their own tests.
```

Comment:
Might be worth adding a special rule in the script since the list of backbones will only increase.

Reply:
Oh thanks, that worked!

Reply:
I'm currently making sure backbones will be tested by the common tests, so will remove this after #20407 is merged